sentry-core/src/main/java/io/sentry/core/protocol/Runtime.java

```diff
@@ -6,7 +6,8 @@
   private String name;
   private String version;
   private String rawDescription;
-  private String build;
+  private String build; // this is not in the documentation
```

Comment:
@bruno-garcia

Reply:
should I remove it?

Reply:
I've added this because we report it in .NET:
https://github.com/getsentry/sentry-dotnet-protocol/blob/afd65b92ed7122a628dc3631711b407f56e9a8a8/src/Sentry.Protocol/Context/Runtime.cs#L40-L45

Another one we should "add to the protocol".

Reply:
I see, but we already have it on Contexts.app.appBuild